caas/kubernetes/provider/application/application.go

```diff
@@ -1106,6 +1109,19 @@ func (a *app) Units() ([]caas.Unit, error) {
 				logger.Tracef("ignoring volume source for projected volume: %v", vol.Name)
 				continue
 			}
+			if vol.ConfigMap != nil {
+				logger.Tracef("ignoring volume source for configMap volume: %v", vol.Name)
+				continue
+			}
+			if vol.HostPath != nil {
+				logger.Tracef("ignoring volume source for hostPath volume: %v", vol.Name)
+				continue
+			}
+			if vol.EmptyDir != nil {
+				logger.Tracef("ignoring volume source for emptyDir volume: %v", vol.Name)
+				continue
```

Comment:
Can we add a small check like is done for secrets - the naming conventions should allow it hopefully

Reply:
I'm not sure how to do this or how the names are generated?